test/lib/buildFactory.test.js

```diff
@@ -254,7 +254,9 @@ describe('Build Factory', () => {
                 environment: { NODE_ENV: 'test', NODE_VERSION: '4' },
                 image: 'node:4',
                 provider: {
-                    name: 'aws'
+                    name: 'aws',
+                    executor: 'sls',
```

Comment:
Are these updated in the doc?

Reply:
Looks like these are already updated. https://docs.screwdriver.cd/cluster-management/aws-integration.html#provider-configuration